packages/react-utils/src/helpers/tests/utils.test.tsx

```diff
@@ -20,7 +20,8 @@ test('date formatting works correctly ', () => {
   jest.spyOn(config, 'getConfig').mockImplementation(() => {
     return { language: undefined };
   });
-  expect(intlFormatDateStrings('01 Jan 1970 00:00:00 GMT')).toEqual('1/1/1970');
+  // fix disparity between circle ci and local tests - ci has no prefixed 0's
```

Comment:
@machariamuguku This is curious!,.. I do not think you should be getting 2 values for this.